radiant/sjra-62/dbt/dbt_project.yml

```diff
@@ -0,0 +1,21 @@
+name: 'sjra62'
+
+config-version: 2
+version: '0.1'
+
+profile: 'starrocks_profile'
+
+test-paths: ["tests"]
+model-paths: ["models"]
+target-path: "target"
+clean-targets:
+    - "target"
+    - "dbt_modules"
+    - "logs"
+
+require-dbt-version: [">=1.0.0", "<2.0.0"]
+
+models:
+  sjra62:
+    staging:
+      materialized: table
```

Comment:
This should change eventually to supported incremental updates with dynamic overwrites to this table.